README.rst

```diff
@@ -1,33 +1,65 @@
-.. Start ignoring LineLengthBear
-
-::
-
-                                                         .o88Oo._
-                                                        d8P         .ooOO8bo._
-                                                        88                  '*Y8bo.
-                                          __            YA                      '*Y8b   __
-                                        ,dPYb,           YA                        68o68**8Oo.
-                                        IP'`Yb            "8D                       *"'    "Y8o
-                                        I8  8I             Y8     'YB                       .8D
-                                        I8  8P             '8               d8'             8D
-                                        I8  8'              8       d8888b          d      AY
-         ,gggo,    ,ggggo,    ,gggo,gg  I8 dP    ,gggo,gg   Y,     d888888         d'  _.oP"
-        dP"  "Yb  dP"  "Y8go*8P"  "Y8I  I8dP    dP"  "Y8I    q.    Y8888P'        d8
-       i8'       i8'    ,8P i8'    ,8I  I8P    i8'    ,8I     "q.  `Y88P'       d8"
-      ,d8,_    _,d8,   ,d8' d8,   ,d8b,,d8b,_ ,d8,   ,d8b,       Y           ,o8P
-    ooP""Y8888PP*"Y8888P"   "Y8888P"`Y88P'"Y88P"Y8888P"`Y8            oooo888P"
-
-.. Stop ignoring LineLengthBear
-
-About
------
+.. image:: https://cloud.githubusercontent.com/assets/7521600/15992701/ef245fd4-30ef-11e6-992d-275c5ca7c3a0.jpg
+
+coala-bears
+-----------
 
 coala-bears is a Python package containing all the bears that are officially
-supported by coala. It features more than **65 bears** covering **35 languages**.
-Here is a `generated list <https://github.com/coala-analyzer/bear-docs/>`_ that
-contains information about each bear, such as the languages it supports and what fixes
-it can apply to your code.
+supported by coala. It features more than **78 bears** covering
+**36 languages**. Here is a `generated list <https://github.com/coala-analyzer/bear-docs/>`_
+that contains information about each bear, such as the languages it supports and
+what fixes it can apply to your code.
+
+-----
+
+.. contents::
+    :local:
+    :depth: 1
+    :backlinks: none
+
+-----
+
+============
+Installation
+============
+
+To install coala-bears, you first need Python 3 or higher. If you do not have it, it is recommended
+that you install Python3 >= 3.4 from `here <https://www.python.org/downloads/>`__.
```

Comment:
Ah didn't see this. Anyway, since already say `3.4, 3.5` with the `|PyPI|` I'm not sure we need this sent